guides/source/active_support_core_extensions.md

```diff
@@ -164,7 +164,13 @@ Active Support provides `duplicable?` to programmatically query an object about
 false.duplicable? # => false
 ```
 
-By definition all objects are `duplicable?` except `nil`, `false`, `true`, symbols, numbers, class, module, and method objects.
+<<<<<<< HEAD
+By definition all objects are `duplicable?` excepting method objects, numbers as well as `nil`, `false`, `true`, symbols, class, and module.
+
+As of Ruby version 2.4.0 numbers, `nil`, `false`, `true`, symbols, class and module objects are also `duplicable?`.
+=======
+By definition all objects are `duplicable?` including BigDecimals, but excepting all other numbers as well as `nil`, `false`, `true`, symbols, class, module, and method objects. As of Ruby 2.4.0 even class and module objects are `duplicable?`.
+>>>>>>> d738fd8... Update list of `duplicable?` objects in AS core_ext guide [ci skip]
```

Comment:
> so I think I'd remove the bit about the general logic of why singletons can not be duplicated.

It's just fine for singletons to return themselves on `dup`, that's how they play the duplicable role. Nothing worth mentioning in this guide, yes.

Merged #27424.